source/administration/mobile-changelog.md

```diff
@@ -32,6 +32,7 @@
  - Fixed an issue where the app performed slowly on Android devices that ran at 120fps instead of the normal 60fps.
 
 ### Known Issues
+ - Custom status hanges from the webapp while the app is in the background do not get reflected on the Mobile App until closing and re-opening the app.
```

Comment:
Custom statuses hang in webapp when the app is in the background is clear; however, the second half of this sentence is less clear. What doesn't get reflected in the Mobile App until users close and reopen?